vendor/bat-native-ledger/src/bat/ledger/internal/promotion/promotion.cc

```diff
@@ -182,9 +182,9 @@ void Promotion::OnGetAllPromotions(
   HandleExpiredPromotions(ledger_, &promotions);
 
   ledger::PromotionList list;
-  bool success = ParseFetchResponse(response, &list);
+  ledger::Result success = ParseFetchResponse(response, &list);
```

Comment:
`success` no longer makes sense as the type is no longer `bool`

Reply:
fixed